filament/include/filament/Skybox.h

```diff
@@ -159,6 +159,20 @@ class UTILS_PUBLIC Skybox : public FilamentAPI {
         */
         Builder& upDirectionAxis(UpDirectionAxis axis) noexcept;
 
+        /** Sets the checkerboard pattern's colors. Both are the level of gray between 0 and 1.
+        * 
+        * Defaults to 1.0 and 0.8.
+        */
+        Builder& checkerboardGrays(math::float2 grays) noexcept;
+
+        /**
+        * Sets the gradient's position and dimming factor. Dimming factor is in X, upper limit
+        * is in Y, location is in Z, and lower limit is in W.
+        * 
+        * Defaults are 0.8, 0.93, 0.55, 0.4.
+        */
+        Builder& gradientSettings(math::float4 settings) noexcept;
```

Comment:
I'd use a struct with named fields instead of encoding it in a float4. Filament uses such structurs often